client/ayon_fusion/api/plugin.py

```diff
@@ -254,11 +252,14 @@ def _get_render_target_enum(self):
             "local": "Local machine rendering",
             "frames": "Use existing frames",
         }
+        default_mode = "local"
         if "farm_rendering" in self.instance_attributes:
             rendering_targets["farm"] = "Farm rendering"
+            default_mode = "farm"
```

Comment:
This seems to make it so that if `Farm Rendering` is enabled in settings that it makes it the default AND also enables allowing to submit it to the farm.

If you remove it there - then suddenly it's not also the default, BUT also the user cannot select it anymore as an option.

As such, we seem to now somehow mix up setting the default value and enabling the toggle itself. Should those be separate?